Resources/Private/Fusion/Prototype/MailObfuscator.fusion

```diff
@@ -18,7 +18,7 @@ prototype(Networkteam.Neos:MailObfuscator) {
     @class = 'Networkteam\\Neos\\MailObfuscator\\Fusion\\ConvertEmailLinksImplementation'
 
     patternMailTo = '/(href=")mailto:([^"]*)/'
-    patternMailDisplay = '/(href="mailto:[^>]*>)([^<]*)/'
+    patternMailDisplay = '|(href="mailto:[^>]*>)(.*)(<\/a>)|'
```

Comment:
This might lead to an unwanted match if two or more a tags are rendered in a row

```html
Email <a href="mailto: test@example.com" itemprop="email"><i class="fa-light fa-paper-plane"></i><span class="btn__text">test@example.com</span></a><a>Some Link</a>
```

(Example: https://www.phpliveregex.com/p/MCl)

I suggest extending the expression to match the content in between with the shortest match possible by using `(.*?)`: `(href="mailto:[^>]*>)(.*?)(<\/a>)`

(Example: https://www.phpliveregex.com/p/MCm)